presto-main/src/main/java/com/facebook/presto/util/MoreMath.java

```diff
@@ -79,6 +81,40 @@ public static double max(double... values)
                 .getAsDouble();
     }
 
+    /**
+     * Find minimum value from finite arguments, and return NaN otherwise.
```

Comment:
Returns the minimum value of the arguments. Returns NaN if there are no arguments or all arguments are NaN.